erpnext/payroll/doctype/payroll_settings/payroll_settings.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Payroll Settings', {
+	// refresh: function(frm) {
```

Comment:
related code should be moved from hr_settings.js

Reply:
done